### DISCUSSION TITLE
Android 第一次播动画会有联网的行为，这个可以去掉

### DISCUSSION THREAD

**Discussion by Git-Boom** [Tencent/libpag#1675] [General] [answered]
var10000 = (HttpsURLConnection)(new URL("https://otheve.beacon.qq.com/analytics/v2_upload")).openConnection();

**Comment by kevingpqi123** [comment, marked as answer]
这个是 pag 的统计上报，当首次使用或更新 pag 版本的时候会触发，个人信息保护规则如下：
https://privacy.qq.com/document/preview/01e79d0cc7a2427ba774b88c6beff0fd

**Comment by dylan4224** [comment]
how to disable or remove this analytics?
